src/app/shared/comcol/sections/comcol-search-section/comcol-search-section.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { Observable } from 'rxjs';
+import { ActivatedRoute, Data } from '@angular/router';
+import { map } from 'rxjs/operators';
+import { SEARCH_CONFIG_SERVICE } from '../../../../my-dspace-page/my-dspace-page.component';
+import { SearchConfigurationService } from '../../../../core/shared/search/search-configuration.service';
+import { RemoteData } from '../../../../core/data/remote-data';
+import { Community } from '../../../../core/shared/community.model';
+import { Collection } from '../../../../core/shared/collection.model';
+import { APP_CONFIG, AppConfig } from '../../../../../config/app-config.interface';
+
+@Component({
+  selector: 'ds-comcol-search-section',
+  templateUrl: './comcol-search-section.component.html',
+  styleUrls: ['./comcol-search-section.component.scss'],
+  providers: [
+    {
+      provide: SEARCH_CONFIG_SERVICE,
+      useClass: SearchConfigurationService,
+    },
+  ],
+})
+export class ComcolSearchSectionComponent implements OnInit {
```

Comment:
Can we add a basic description of this component in a comment at the top of the class?  It's just useful for other developers to immediately know where/how it is being used